src/args.rs

```diff
@@ -1,8 +1,125 @@
-#[derive(Debug)]
-pub struct Args {}
+use clap::Parser;
+
+#[derive(Parser, Debug, Default)]
+#[command(author, version, about, long_about = None, name = "jperf")]
+pub struct Args {
+	/// Application name
+	#[arg(long)]
+	app: Option<String>,
```

Comment:
These comments would nicely be expressed by using `value_name` in the `arg` attribute, this would then free us up for a more descriptive doc string, perhaps like this:
```rust
/// Filter results to just one app
#[arg(long, value_name = "name")]
app: Option<String>,
```

Reply:
I've added `value_name` to each, and tried to keep them as a short descriptive test, but unsure if I did it properly as it should be - could you pls check and let me know what you think.

Reply:
Ah, I see the meaning of value_name better now: basically the argument flag and the `value_name` together have a concise and descriptive meaning. The word "name" does make perfect sense. I was considering `value_name`  as some sort of tag, and wanted to give each of them a different one; but I now understand what it's intended for.
Went for `milliseconds`.